src/app/components/Book/Book.jsx

```diff
@@ -1,29 +1,70 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import config from '../../../../appConfig.js';
+import { EReaderIcon, BookIcon } from 'dgx-svg-icons';
+import config from '../../../../appConfig';
 import utils from '../../utils/utils';
 
 const Book = ({ book }) => {
-  const fullImgSrc = (!book.imageSlug || book.imageSlug === '') ? `${config.baseUrl}src/client/images/book-place-holder.png` :
-    'https://contentcafe2.btol.com/ContentCafe/Jacket.aspx' +
-    `?&userID=NYPL49807&password=CC68707&Value=${book.imageSlug}&content=M&Return=1&Type=M`;
   const tagArray = book.tags.map(tag => tag.toLowerCase().split(' ').join('-'));
   const tagClasses = tagArray.join(' ');
 
+  const renderBookCoverImage = (imageUrl) => {
+    const defaultImageUrl = `${config.baseUrl}src/client/images/book-place-holder.png`;
+    const fullImgSrc = (!imageUrl || imageUrl === '') ? defaultImageUrl : imageUrl;
+    return (
+      <div className="book-item-image-box">
+        <img alt="" src={fullImgSrc} />
+      </div>
+    );
+  };
+
+  const renderTitle = (title, link) => {
+    const titleClass = 'book-item-title';
+
+    if (title && title !== '') {
```

Comment:
You should be able to just do `if (title) {` since `''` will return a falsy value.

Reply:
what if title is not of type string, possibly an object or array? maybe I should simply check typeof? I guess I had issues like this with the Hold stuff and needed to be explicit about empty strings... 

```
if (typeof title === 'string' && title.trim() !== '') // better?
```